spec/mongoid/persistable/savable_spec.rb

```diff
@@ -83,6 +83,27 @@
             end
           end
         end
+
+        context 'when the embedded document is unchanged' do
+
+          before do
+            allow(Kangaroo.collection).to receive(:insert).once
```

Comment:
I agree here as well, since the expectation is the actual test here, not that the call to save is true.